asdf/_tests/_helpers.py

```diff
@@ -286,57 +275,6 @@ def _assert_roundtrip_tree(
             asdf_check_func(ff)
 
 
-def yaml_to_asdf(yaml_content, yaml_headers=True, standard_version=None):
```

Comment:
This was replaced with `asdf.testing.helpers.yaml_to_asdf` instead of updating it to remove the use of `version_map`.